backend/gzdb/feeds.go

```diff
@@ -207,8 +211,9 @@ func dbPollToZeniPoll(poll *Poll) (*zeni.Poll, error) {
 		})
 		for _, user := range result.Users {
 			votes = append(votes, &zeni.Vote{
-				UserID: strconv.FormatUint(uint64(user.ID), 10),
-				Option: result.Option,
+				CreatedAt: user.CreatedAt,
```

Comment:
mmh, so the vote timestamp can be before the poll creation timestamp?

I think we should use `result.CreatedAt` here, and later rework the db to have a `poll_options` and `poll_votes` tables instead of `poll_results` (which is actually options) and implicit `poll_votes` created by the many2many without timestamps